web/sites/default/config/google_tag.settings.yml

```diff
@@ -20,19 +35,4 @@ _default_container:
   status_toggle: 'exclude listed'
   status_list: |-
     403
-    404
-  data_layer: dataLayer
-  include_classes: false
-  whitelist_classes: |-
-    google
-    nonGooglePixels
-    nonGoogleScripts
-    nonGoogleIframes
-  blacklist_classes: |-
-    customScripts
-    customPixels
-  include_environment: false
-  environment_id: ''
-  environment_token: ''
-_core:
-  default_config_hash: 65d5DfK5p2hQDpBskbvCIj2hSEqw_Odm2e_d46BH1Lk
+    404
```

Comment:
File is missing newline.

Reply:
Since this is generated by Drupal, I'm not sure we should actually add a newline. 🤔